crates/matrix-sdk/src/event_cache/room/events.rs

```diff
@@ -231,6 +255,35 @@ impl RoomEvents {
         self.chunks.items()
     }
 
+    /// Return the order of an event in the room (main) linked chunk.
```

Comment:
You can remove “(main)” I think:

- it can only create confusion,
- to me, it's clear that a room has a linked chunk, and that each thread will have its own linked chunk,
- you are saying “in the room”, so we know it's not for the thread.

The comment is already crystal clear, you don't need “(main)”.